core/include/scipp/core/flags.h

```diff
@@ -8,6 +8,8 @@
 
 namespace scipp {
 
+enum class SCIPP_CORE_EXPORT CopyPolicy { Always, TryAvoid };
```

Comment:
`TryAvoid` is a bit hard to remember. How about just `Avoid`?

Reply:
I wanted to highlight that it may still copy.